package.json

```diff
@@ -23,17 +23,16 @@
     "test": "npm run prettier:check && npm run xo"
   },
   "devDependencies": {
-    "autoprefixer": "^9.8.6",
+    "autoprefixer": "^10.0.4",
     "eslint-plugin-compat": "^3.8.0",
-    "postcss-cli": "^7.1.2",
+    "postcss": "^8.1.10",
+    "postcss-cli": "^8.3.0",
     "prettier": "2.2.1",
     "xo": "^0.35.0"
   },
   "browserslist": [
     "defaults",
-    "Explorer >= 11",
-    "not ExplorerMobile <= 11",
-    "not Safari 5.1"
+    "Explorer >= 11"
```

Comment:
Worth noting that we're going to stop trying to support IE11 in future:

From most recent release notes:

![image](https://user-images.githubusercontent.com/1998970/100540880-1e217a80-3238-11eb-81da-2eba5653d1f7.png)